webdriver/tests/bidi/input/set_files/set_files.py

```diff
@@ -0,0 +1,273 @@
+import pytest
+
+from .. import get_events
+from webdriver.bidi.error import NoSuchElementException, UnableToSetFileInputException
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_set_files(
+    bidi_session,
+    top_context,
+    load_static_test_page,
+    get_element,
+):
+    await load_static_test_page(page="files.html")
+
+    element = await get_element("#input")
+
+    await bidi_session.input.set_files(
+        context=top_context["context"],
+        element={"sharedId": element["sharedId"]},
```

Comment:
I think we can do this everywhere.
```suggestion
        element=element,
```